app/controllers/ncr/work_orders_controller.rb

```diff
@@ -4,12 +4,11 @@ class WorkOrdersController < UseCaseController
     MAX_UPLOADS_ON_NEW = 10
 
     def new
-      @approver_email = self.suggested_approver_email
+      @model_instance.approving_official_email = self.suggested_approver_email
```

Comment:
is this controller being used for more than one type of object? of not, I think something like `@ncr_work_order` would be a clearer var name than `@model_instance`